google/earthenterprise
# Fixes #1105:  turn world-writable files back into symlinks

Fixes #1105 

Turn world writable files back into symlinks in the rpm post-install script. Gradle resolves symlinks instead of preserving them, generating world-writable files instead of symlinks. This resolves this issue until the issue has been fixed in gradle or an alternative solution is found. This affects all platforms, so it must be tested on all of them (Centos 6/7, RHEL 6/7, Ubuntu 14/16).

To test:

1. Build OpenGEE RPMs. See the build [instructions](https://github.com/google/earthenterprise/blob/master/earth_enterprise/BUILD.md). The following command does all the building:
`scons -j8 release=1 build; scons -j8 release=1 stage_install; scons -j8 release=1 package_install`

2. Install those RPMs using yum (or dpkg on Ubuntu).

3. `sudo find /opt/google/ -type f -exec ls -l {} \; | grep rwxrwxrwx`

4. Verify there is no output.
